doc/classes/TileMap.xml

```diff
@@ -4,7 +4,7 @@
 		Node for 2D tile-based maps.
 	</brief_description>
 	<description>
-		Node for 2D tile-based maps. Tilemaps use a [TileSet] which contain a list of tiles which are used to create grid-based maps. A TileMap may have several layers, layouting tiles on top of each other.
+		Node for 2D tile-based maps. Tilemaps use a [TileSet] which contain a list of tiles which are used to create grid-based maps. A TileMap may have several layers layering tiles on top of each other.
```

Comment:
```suggestion
		Node for 2D tile-based maps. Tilemaps use a [TileSet] which contain a list of tiles which are used to create grid-based maps. A TileMap may have several layers, allowing to stack different tiles on top of each other.
```

Reply:
"allowing to stack different tiles on top of each other" is also incorrect.

Reply:
How so? You said yourself that what was described here was stacking.

> Laying out or "layout" typically refers to an arrangement of items in close proximity, not stacked like in this situation. It might be better to simplify and say that a Tilemap may contain several layers of tiles on top of each other.

I just paraphrased your suggestion.

Reply:
You actually both changed the meaning of my suggestion (the layers themselves are on top of each other, not just the tiles) and made it longer (replacing "of" with "allowing to stack different"), so that's not paraphrasing. Further, if you're using "allowing" in a sentence, you also have to explain who or what is allowed like "allowing you" or "allowing the user."

Reply:
Fair enough, I won't try to improve that one and restore the previous suggestion.